private-templates-service/server/src/app.ts

```diff
@@ -20,6 +22,19 @@ app.use(passport.initialize());
 app.use(passport.session());
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
+app.use(helmet.hsts({
+  maxAge: 15552000,
+  includeSubDomains: false
```

Comment:
Don't disable subDomains. We do not use any for now, but if somebody will want to host some part of the website on subdomain then hsts won't be enabled there :)

Reply:
Also, please remove **package.json** from the PR. 
run **git checkout origin/dev -- package.json** in the correct directory